examples/.railway/next-sse-chat/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN corepack enable
 
 ARG DATABASE_URL
 ENV DATABASE_URL=$DATABASE_URL
+ARG RAILWAY_ENVIRONMENT_NAME
+ENV RAILWAY_ENVIRONMENT_NAME=$RAILWAY_ENVIRONMENT_NAME
```

Comment:
is this needed? we need the database_url here cause of it running migrations but i thought railway injected their stuff automatically?